kalite/control_panel/tests/control_panel.py

```diff
@@ -107,6 +107,34 @@ def test_teachers_have_no_facility_delete_button(self):
         with self.assertRaises(NoSuchElementException):
             self.browser.find_element_by_xpath('//a[@class="facility-delete-link"]')
 
+    def test_facility_with_no_missing_metadata(self):
+        facility_name = 'no-missing-metadata'
+        self.fac = self.create_facility(name=facility_name)
```

Comment:
I think this is the cause of your test failure - you are creating a facility here, but with no more metadata than in your 'no_metadata' test.

Fill out all the fields that you name in your 'check_metadata' function when you initialize this facility, and it should work!